sniffer/main.c

```diff
@@ -37,6 +37,11 @@
  */
 #define RAWDUMP_PRIO            (THREAD_PRIORITY_MAIN - 1)
 
+/**
+ * @brief   Message queue size of the RAW dump thread
+ */
+#define RAWDUMP_MSG_Q_SIZE      (32)
```

Comment:
this looks rather huge, don't you think? Minor-nit besides that: use `(32U)`?

Reply:
Will do the `32U`. Considering, that not loosing messages is the whole point of this endeavour I think this isn't huge (also the application should have enough space, the app clocks in at 11.5 KB RAM at the moment).